exec.go

```diff
@@ -24,7 +24,8 @@ type EXEC struct {
 
 // CommandOptions contains the options that can be passed to command.
 type CommandOptions struct {
-	Dir string
+	Dir             string
+	Continueonerror bool
```

Comment:
The js serializer seems to have difficulty with camel case names, so only the leading letter could be capitalized.

Reply:
Maybe this helps https://k6.io/docs/extensions/explanations/go-js-bridge/

Reply:
Thanks so much! Updated the PR.